src/Tests/DIPS.Xamarin.UI.Tests/Converters/ValueConverters/DateAndTimeConverterTests.cs

```diff
@@ -70,7 +70,7 @@ public void Convert_WithShortFormat_WithCulture_CorrectFormat(string cultureName
                 new object[] { "no", new DateTime(1990, 12, 10, 09, 09, 00), "10. des kl 09:09" }
             };
 
-        [Theory]
+        [Theory(Skip="Issue with region")]

```

Comment:
Remove skip, this will be fixed when this is merged: https://github.com/DIPSAS/DIPS.Xamarin.UI/pull/208

Reply:
Done